.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -91,4 +84,20 @@ Try it out yourself:
 
 Interested in playing around with the API? Documentation is [here](https://cataas.com/#/).
 
-## **:camping: To move on, comment your cat picture 🐱**
+
+### How to Navigate Postman's UI
+//insert gif of sending a request
+
+<details>
+  <summary>:question: How do we build requests?</summary>
```

Comment:
This needs to be a step by step explination on how to use postman to send the cat api request with screenshots